tests/models/test_experts_int8.py

```diff
@@ -0,0 +1,25 @@
+# flake8: noqa
+"""Tests experts_int8 quantization startup and generation, 
+doesn't test correctness
+"""
+
+import pytest
+
+MODELS = ["ai21labs/Jamba-tiny-random"]
+
+
+@pytest.mark.parametrize("model", MODELS)
```

Comment:
This should be disabled on GPUs that can't support the quant method
```python
from tests.quantization.utils import is_quant_method_supported

@pytest.mark.skipif(not is_quant_method_supported("experts_int8"),
                    reason="ExpertsInt8 is not supported on this GPU type.")
```

Reply:
Addressed